### ISSUE TITLE
Error while compiling: Install Targets given no Library Destionation for shared library target

### ISSUE THREAD

**Issue by Gandalf1783** [mitsuba-renderer/nanogui#125]
I cannot compile the current version of nanogui.

After I ran the cmake command, this came out:
```
root@project:~/nanogui# cmake ./
-- NanoGUI v0.1.4
-- The CXX compiler identification is GNU 8.3.0
-- The C compiler identification is GNU 8.3.0
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Setting build type to 'Release' as none was specified.
-- NanoGUI: using OpenGL backend.
-- NanoGUI: building shared library.
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Looking for pthread_create
-- Looking for pthread_create - not found
-- Looking for pthread_create in pthreads
-- Looking for pthread_create in pthreads - not found
-- Looking for pthread_create in pthread
-- Looking for pthread_create in pthread - found
-- Found Threads: TRUE
-- Using X11 for window creation
-- Looking for XOpenDisplay in /usr/lib/x86_64-linux-gnu/libX11.so;/usr/lib/x86_64-linux-gnu/libXext.so
-- Looking for XOpenDisplay in /usr/lib/x86_64-linux-gnu/libX11.so;/usr/lib/x86_64-linux-gnu/libXext.so - found
-- Looking for gethostbyname
-- Looking for gethostbyname - found
-- Looking for connect
-- Looking for connect - found
-- Looking for remove
-- Looking for remove - found
-- Looking for shmat
-- Looking for shmat - found
-- Looking for IceConnectionNumber in ICE
-- Looking for IceConnectionNumber in ICE - found
-- Found X11: /usr/lib/x86_64-linux-gnu/libX11.so
CMake Error at CMakeLists.txt:521 (install):
  install TARGETS given no LIBRARY DESTINATION for shared library target
  "nanogui".


CMake Error at /usr/share/cmake-3.13/Modules/WriteBasicConfigVersionFile.cmake:30 (message):
  Unknown keywords given to WRITE_BASIC_CONFIG_VERSION_FILE():
  "ARCH_INDEPENDENT"
Call Stack (most recent call first):
  /usr/share/cmake-3.13/Modules/CMakePackageConfigHelpers.cmake:211 (write_basic_config_version_file)
  CMakeLists.txt:529 (write_basic_package_version_file)


-- Configuring incomplete, errors occurred!
See also "/root/nanogui/CMakeFiles/CMakeOutput.log".
See also "/root/nanogui/CMakeFiles/CMakeError.log".
root@project:~/nanogui#
```

It seems like an error in CMakeList.txt?
Im not familiar with cmake, so I have no clue to solving this myself. 

Running this on Debian 4.19.208-1 / x86_64.

**Comment by Gandalf1783** [comment]
I just found the docs, but even with a build folder, this will stil result in this:
```
root@project:~/nanogui/build# cmake ..
-- NanoGUI v0.1.4
-- NanoGUI: using OpenGL backend.
-- NanoGUI: building shared library.
-- Using X11 for window creation
CMake Error at CMakeLists.txt:521 (install):
  install TARGETS given no LIBRARY DESTINATION for shared library target
  "nanogui".


CMake Error at /usr/share/cmake-3.13/Modules/WriteBasicConfigVersionFile.cmake:30 (message):
  Unknown keywords given to WRITE_BASIC_CONFIG_VERSION_FILE():
  "ARCH_INDEPENDENT"
Call Stack (most recent call first):
  /usr/share/cmake-3.13/Modules/CMakePackageConfigHelpers.cmake:211 (write_basic_config_version_file)
  CMakeLists.txt:529 (write_basic_package_version_file)


-- Configuring incomplete, errors occurred!
See also "/root/nanogui/CMakeFiles/CMakeOutput.log".
See also "/root/nanogui/CMakeFiles/CMakeError.log".
root@project:~/nanogui/build#
```

**Comment by wjakob** [comment]
Should be fixed in fdf6671, please reopen if it still causes problems.
